doc/grade-data.md

```
@@ -16,11 +16,15 @@ $ uv run python fetch_grades.py <year>_<season>
 Output saved in `tcf_website/management/commands/grade_data/csv`
 
 ## Loading Grade Data
-- To load grades, run _in the docker container_:
+
+With the local Compose stack running, load grades into PostgreSQL with:
+
 ```console
-$ python manage.py load_grades ALL_DANGEROUS
+$ docker compose --profile full exec web python manage.py load_grades ALL_DANGEROUS
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)